src/MICore/UnixUtilities.cs

```diff
@@ -160,5 +163,28 @@ internal static bool IsProcessRunning(int processId)
             // if there is no process with the ID specified.
             return UnixNativeMethods.GetPGid(processId) >= 0;
         }
+
+        public static bool IsBinarySigned(string filePath)
```

Comment:
Would it make sense to cache this to avoid needing to re-check the debugger every time?